<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMUCostumeProfileEditionWizard</class>
 <widget class="QWizard" name="IMUCostumeProfileEditionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>783</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Costume Profile Edition Wizard - %1 [*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../imu_view/ui/MdeViewIcons.qrc">
    <normaloff>:/mde/icons/imu.png</normaloff>:/mde/icons/imu.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="modelConfigurationPage">
   <property name="title">
    <string>Model configuration</string>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="modelClassLabel">
      <property name="text">
       <string>Class:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="modelClassComboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="modelLabel">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="modelComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="createModelPushButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>504</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="loadModelPushButton">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="modelPhisicalDimensionsPage">
   <property name="minimumSize">
    <size>
     <width>525</width>
     <height>510</height>
    </size>
   </property>
   <property name="title">
    <string>Model physical dimensions</string>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <row>
         <property name="text">
          <string>Body Height</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Foot Size</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Ankle Height</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Hip Height</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Hip Width</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Knee Height</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Shoulder Width</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Shoe Sole Height</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value [m]</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1,85</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0,26</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0,12</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0,89</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>0,34</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>0,32</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>0,43</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string>0,04</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="bodyDimensionsLabel">
      <property name="minimumSize">
       <size>
        <width>375</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>375</width>
        <height>500</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="imuCostumeIcons.qrc">:/imuCostume/icons/skeletonWithDimensions.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="sensorsConfigurationPage">
   <property name="title">
    <string>Sensors configuration</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTableWidget" name="sensorsConfigurationTableWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Sensor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Joint</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estimation Algorithm</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rotation</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Offset</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="oneToOneMappingCheckBox">
          <property name="text">
           <string>One-To-One mapping</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="motionEstimationCalibrationPage">
   <property name="title">
    <string>Motion estimation configuration</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="calibrationAlgorithLabel">
          <property name="text">
           <string>Calibration algorithm:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="calibrationAlgorithmComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="motionEstimationAglorithmLabel">
          <property name="text">
           <string>Motion estimation algorithm:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="motionEstimationAglorithmComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>546</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../imu_view/ui/MdeViewIcons.qrc"/>
  <include location="imuCostumeIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>IMUCostumeProfileEditionWizard</sender>
   <signal>currentIdChanged(int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>pageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensorsConfigurationTableWidget</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>onSensorConfigurationCellChange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modelComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>verifyModel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibrationAlgorithmComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>verifyCalibrationAlgorithm(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneToOneMappingCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>onOneToOneMappingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motionEstimationAglorithmComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>verifyMotionEstimationAlgorithm(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadModelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>onLoadModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensorsConfigurationTableWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>IMUCostumeProfileEditionWizard</receiver>
   <slot>onSensorsConfigurationTableContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pageChanged(int)</slot>
  <slot>verifyCalibrationAlgorithm(int)</slot>
  <slot>verifyMotionEstimationAlgorithm(int)</slot>
  <slot>onSensorConfigurationCellChange(int,int)</slot>
  <slot>verifyModel(int)</slot>
  <slot>onCellPressed(int,int)</slot>
  <slot>onCellActivated(QModelIndex)</slot>
  <slot>onLoadModel()</slot>
  <slot>onSensorsConfigurationTableContextMenu(QPoint)</slot>
 </slots>
</ui>
